en/mulesoft/modules/ROOT/pages/ms_composer_http_reference.adoc

```diff
@@ -0,0 +1,117 @@
+= HTTP Connector Reference
+
+The HTTP Connector connects cloud applications that are not currently supported through dedicated connectors. The HTTP Connector allows you to connect to systems that Composer does not currently support to extend the capabilities of the systems that Composer does support. The HTTP Connector for Composer uses https://www.digitalocean.com/community/tutorials/http-1-1-vs-http-2-what-s-the-difference#http-1-1[HTTP] 1.1 to handle JSON data.
+
+== Connections
+
+To connect to a system, whether it is a source of data or the target where you are sending data, you must create a connection with that system's required credentials. You typically connect to different systems within a flow and can reuse each connection every time you need to connect.
```

Comment:
```suggestion
To connect to a system, whether it is a source of data or the target where you are sending data, you must create a connection with the required credentials of that system. You typically connect to different systems within a flow and can reuse each connection every time you need to connect.
```

Reply:
I don't know if this is necessary or not, I just tend not to use possessives in this way. Check the style guide?